Src/Core/Equivalency/FieldSelectedMemberInfo.cs

```diff
@@ -30,5 +31,15 @@ public override Type MemberType
         {
             get { return fieldInfo.FieldType; }
         }
+
+        public override CSharpAccessModifier GetAccessModifier
```

Comment:
* Well, if you would state the same thing in English, you would say something like `which setter has modifier Internal`.
* WhichSetterDoesNotHaveModifier. On second thought, maybe remove the `Modifier` suffix from all methods.  
* I would remove the public property.